src/utility/copy.cpp

```diff
@@ -135,7 +140,7 @@ appendCreateCopyOptions(StringInfo info, CopyStmt *copy_stmt) {
 			switch (nodeTag(defel->arg)) {
 			case T_Integer:
 			case T_Float:
-			case T_Boolean:
```

Comment:
This change is incorrect, `T_Boolean` and `T_BooleanTest` are very different nodes. The `Boolean` node type did not exist in PG14, it was using an integer or a string before so we can simply skip it there.

Reply:
It's my fault.  Sorry, I didn't notice a usage error.  It was very careless of me.  Thank you for your correction.